core/src/test/java/org/opensearch/sql/expression/function/OpenSearchFunctionsTest.java

```diff
@@ -154,6 +154,14 @@ void match_to_string() {
     assertEquals("match(field=\"message\", query=\"search query\")", expr.toString());
   }
 
+  @Test
+  void multi_match() {
+    FunctionExpression expr = dsl.multi_match(fields, query);
+    assertEquals(String.format("multi_match(fields=%s, query=%s)",
+            fields.getValue().toString(), query.getValue().toString()),
```

Comment:
There's no need to call `toString` in argument to String.format.

It will be called automatically -- https://docs.oracle.com/javase/7/docs/api/java/util/Formatter.html